DataFormats/L1TGlobal/test/TestGlobalObjectMapRecordFormat.sh

```diff
@@ -10,7 +10,23 @@ file=testGlobalObjectMapRecord.root
 
 cmsRun ${LOCAL_TEST_DIR}/test_readGlobalObjectMapRecord_cfg.py "$file" || die "Failure using test_readGlobalObjectMapRecord_cfg.py $file" $?
 
-oldFiles="testGlobalObjectMapRecord_CMSSW_13_0_0.root testGlobalObjectMapRecord_CMSSW_13_1_0_pre3.root"
+# The old files read below were generated as follows.
+#
+#     Check out the release in the filename and cherry pick the commit that
+#     adds the original version of the file:
+#     DataFormats/L1TGlobal/test/TestWriteGlobalObjectMapRecord.cc
+#
+# Run cmsRun with DataFormats/L1TGlobal/test/create_GlobalObjectMapRecord_test_file_cfg.py
+# as the configuration and rename the file that creates.
+#
+# By default, split level 99 is used (maximum possible splitting).
+# If the suffix "_split_0" is near the end of the filename, the
+# following was added to the configuration of the output module:
+#     "splitLevel = cms.untracked.int32(0)"
+
+oldFiles="testGlobalObjectMapRecord_CMSSW_13_0_0_split_99.root testGlobalObjectMapRecord_CMSSW_13_1_0_pre3_split_99.root"
+oldFiles+=" testGlobalObjectMapRecord_CMSSW_13_0_0_split_0.root"
+oldFiles+=" testGlobalObjectMapRecord_CMSSW_13_1_0_pre3_split_0.root"
```

Comment:
How about reorganizing along
```suggestion
oldFiles="testGlobalObjectMapRecord_CMSSW_13_0_0_split_99.root testGlobalObjectMapRecord_CMSSW_13_0_0_split_0.root"
oldFiles+=" testGlobalObjectMapRecord_CMSSW_13_1_0_pre3_split_99.root testGlobalObjectMapRecord_CMSSW_13_1_0_pre3_split_0.root"
```
? If this data format will ever be changed again, there would be a clear pattern on adding the files corresponding to a new release.

Reply:
Done. Looks better. Thanks.